lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -778,17 +761,60 @@ def get_file(file, use_token: true)
         test_output_against_profile('Procedure', profile_definitions)
       end
 
-      test :validate_medication do
+      test :validate_encounter do
+        metadata do
+          id '18'
+          name 'Encounter resources on the FHIR server follow the US Core Implementation Guide'
+          link 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-encounter'
+          description %(
+            This test checks if the resources returned from bulk data export conform to the US Core profiles. This includes checking for missing data elements and valueset verification.
+
+            Encounter is membership resource of [FHIR Patient Compartment](https://www.hl7.org/fhir/compartmentdefinition-patient.html).
```

Comment:
```suggestion
```

What is the purpose of this statement?

Reply:
Bulk data export include resources from FHIR Patient Compartment

Reply:
```suggestion
            Encounter is a membership resource of [FHIR Patient Compartment](https://www.hl7.org/fhir/compartmentdefinition-patient.html).
```

minor grammer fix